Me: Add new section, "Get Apps"

jasmussen opened the pull request:
This PR aims to add a new subsection under the me tag, subheadline "Special", labelled "Get Apps". The purpose is to create one central place to reference desktop and mobile application clients for WordPress and WordPress.com.

Mockup:

![mockup](https://cloud.githubusercontent.com/assets/1204802/12294324/b8c5b2de-b9f9-11e5-91f2-16b90f368e66.png)

Screenshot of current status:

![screenshot](https://cloud.githubusercontent.com/assets/1204802/12294325/bbb73ec2-b9f9-11e5-9cc2-1957b03d84b7.png)

Before I proceed, I would like feedback on this PR as-is. Both on content, design, and code, thank you. Depending on feedback, at some point I'll need to add a few images to this section, SVG or PNG in the case of app store link buttons. What would be the best approach for adding these images?

mtias commented on client/me/apps/README.md:
```diff
@@ -0,0 +1,4 @@
+Apps
```
I'd call the component `get-apps` for a bit more clarity.

mtias commented on client/me/apps/style.scss:
```diff
@@ -0,0 +1,35 @@
+.apps .apps__title {
```
It shouldn't need the extra level of specificity, just `.apps__title` would be fine.

mtias commented on client/me/apps/style.scss:
```diff
@@ -0,0 +1,35 @@
+.apps .apps__title {
+	text-align: center;
+	font-size: 24px;
+	font-family: Merriweather, Georgia, "Times New Roman", Times, serif;
```
There's a `$serif` var to output the font stack.

jasmussen commented:
Addressed your feedback, Matias, superb as always thanks a bunch for doing this. Also rebased on master. Will start to add images now, and put them in `/public/images`.

mtias commented on assets/stylesheets/_components.scss:
```diff
@@ -121,6 +121,7 @@
 @import 'me/action-remove/style';
 @import 'me/application-password-item/style';
 @import 'me/application-passwords/style';
+@import 'me/get-apps/style';
```
small nitpick, but we should now move it to forwards for the "g" letter. :)

mtias commented on client/me/get-apps/index.jsx:
```diff
@@ -0,0 +1,79 @@
+						<div className="get-apps__app-icon">
+						</div>
+						<p><strong>Mac</strong><br />
+						Requires 10.11 or newer</p>
```
We should probably translate all these as well?

mtias commented:
My pleasure! Looking good.

jasmussen commented:
I think this is ready for final review. It looks like this now:

![screen shot 2016-01-14 at 12 46 44](https://cloud.githubusercontent.com/assets/1204802/12323593/096cfbfc-babd-11e5-9ed6-cf2d3aec4ec0.png)

mjangda commented on client/me/get-apps/index.jsx:
```diff
@@ -0,0 +1,84 @@
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+							<img src="/calypso/images/me/get-apps-osx.svg" alt="" width="96" height="96" />
```
Should we fill in the `alt` attributes for the logos?

mjangda commented on client/me/get-apps/index.jsx:
```diff
@@ -0,0 +1,84 @@
+							<img src="/calypso/images/me/get-apps-ios.svg" alt="" width="96" height="96" />
+						</div>
+						<p><strong>{ this.translate( 'iOS' ) }</strong><br />
+						{ this.translate( 'Requires 9 or newer' ) }</p>
```
I wonder if we should drop the `requires` for the mobile apps. They could end up changing and we'd need to remember to update this page. Plus, the app store links will already include this info.

mjangda commented on client/me/get-apps/index.jsx:
```diff
@@ -0,0 +1,84 @@
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+							<img src="/calypso/images/me/get-apps-windows.svg" alt="" width="96" height="96" />
```
It might be nice to linkify the the logos as well?

mjangda commented on client/me/get-apps/index.jsx:
```diff
@@ -0,0 +1,84 @@
+						</div>
+						<p><strong>{ this.translate( 'iOS' ) }</strong><br />
+						{ this.translate( 'Requires 9 or newer' ) }</p>
+						<a href="https://itunes.apple.com/us/app/wordpress/id335703880?mt=8"><img src="/calypso/images/me/get-apps-app-store.png" alt="" /></a>
```
We should definitely have `alt` for the App Store and Google Play logos since they are linked and don't have text.

mjangda commented:
Looks pretty solid! Added some minor comments.

Also, noticed that on the smaller breakpoints the page title gets flushed a little too close to the nav bar:

![get_apps_ _wordpress_com](https://cloud.githubusercontent.com/assets/86105/12333580/00fb88fc-bac3-11e5-8507-3db84c600609.png)

mjangda commented:
Oh, one more thing: for the Desktop download links, can we add `?ref=getapps` to the URL?

jasmussen commented:
Added alt tags everywhere, added some topmargin on the mobile breakpoint, and added the ref to the download links. Did not linkify the logos. There are some relatively strict guidelines for how to use the app store badges, and in the case of Linux, we have two buttons, might even have three in the future.

mjangda commented:
:+1:
